static/style.css

```diff
@@ -355,18 +369,115 @@
   margin-right: 12px;
 }
 
-.buttons a:hover {
+.contact .form button {
+  height: 48px;
+  background-color: #ff6b00;
+  padding: 12px;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  border: 0;
+}
+
+.buttons a:hover,
+.contact .button button:hover {
   color: #f55800;
   background-color: #fff5e1;
   border: #f55800 solid 1px;
 }
 
-.buttons a:active {
+.buttons a:active,
+.contact .button button:active {
   color: #fff;
   background-color: #e05304;
 }
 
-.buttons a:disabled {
+.buttons a:disabled,
+.contact .button button:disabled {
   color: #5b5957;
   background-color: #f4f1f0;
 }
+
+.contact {
+  height: 700px;
+  margin-top: 69px;
+  border-bottom: #dfe1e6 solid 1px;
+}
+
+.contact .announcement p {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact .announcement {
+  margin-left: 23px;
+  margin-right: 25px;
+}
+
+.contact .form {
+  margin-left: 23px;
+  margin-right: 25px;
+  margin-top: 100px;
+}
```

Comment:
- [ ] Please use Flexbox for your form because it will make work easier to align the content properly after removing the unordered list tag your design need some touch up which will be easier by using Flex. Kindly adhere to the project requirement and do implement the things in your project before submitting it for another review.

### Requirement

![image](https://github.com/wacoo/Portfolio-Project/assets/66476329/75bb39a8-8518-4c9e-8db9-f77baf41bd16)